public/about/team/index.html

```diff
@@ -176,96 +163,147 @@ <h5 class="card-title ">Elena Izotova</h5>
             <table>
               <tr>
                 <td>
-                  <img class="team-photo " src="/img/team/vrinda_punj.jpg ">
+                  <img class="team-photo " src="/img/team/male.png ">
                 </td>
                 <td>
-                  <h5 class="card-title ">Vrinda Punj</h5>
+                  <h5 class="card-title ">Samuel Zuk</h5>
                   <p class="team-title ">
                     Developer
                   </p>
                   <p class="card-text ">
-                    <div class="team-degree ">B.S. Computer Science 2021</div>
+                    <div class="team-degree ">B.S. Computer Science 2022</div>
+                  </p>
+                </td>
+            </table>
+          </div>
+        </div>
+        <div class="card info-card col-lg-5 col-sm-12 mr-auto ">
+          <div class="card-body ">
+            <table>
+              <tr>
+                <td>
+                  <img class="team-photo " src="/img/team/chris_berns.jpg ">
```

Comment:
Did you intend to use Chris's photo for your photo?

Reply:
Ah, no, my bad, I initially changed it to the default, but when copying some template stuff around I must have accidentally overwritten that change.